### ISSUE TITLE
My network is not generalizing - please help

### ISSUE THREAD

**Issue by amantalion** [nkrao220/capstone-2#1]
Hi @nkrao220,

My network (moz_cnn_2.py) is not generalizing at all. The validation accuracy goes up by 2-3% and is stuck at 50%. Please, could you give me some pointers on how you trained yours and forced the validation accuracy to go up?

Also, one thing that I do differently: I start training directly on the Common Voice dataset without pre-training on the smaller dataset.

**Comment by klo0003** [comment]
@amantalion I realize this question was posted a while ago, but were you able to solve this problem? I'm having the same issue.
